cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala

```diff
@@ -204,6 +204,7 @@ m|LONG
 |a|The initialization stacktrace for this transaction or an empty string if unavailable.
 |m|STRING
 ||===""")
+      p("The `SHOW TRANSACTIONS` command can be combined with multiple `SHOW TRANSACTIONS` and `TERMINATE TRANSACTIONS`, see <<query-combine-tx-commands, transaction commands combination>>.")
```

Comment:
```suggestion
      p("The `SHOW TRANSACTIONS` command can be combined with multiple `SHOW TRANSACTIONS` and `TERMINATE TRANSACTIONS`.
      See <<query-combine-tx-commands, transaction commands combination>>.")
```

Reply:
This isn't a multiline string but I can update it manually to do the change

Reply:
To me the breaking is kinda weird anyway. The second sentence will just be `See <link>.` which doesn't feel like a full sentence at all